src/features/farming/bakery/components/Cakes.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useContext, useState } from "react";
+import Decimal from "decimal.js-light";
+
+import token from "assets/icons/token.gif";
+
+import { Box } from "components/ui/Box";
+import { OuterPanel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+import { Context } from "features/game/GameProvider";
+
+import { useActor } from "@xstate/react";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { ToastContext } from "features/game/toast/ToastQueueProvider";
+import { CAKES, Craftable } from "features/game/types/craftables";
+
+export const Cakes: React.FC = () => {
+  const [selected, setSelected] = useState<Craftable>(
+    CAKES()["Sunflower Cake"]
+  );
+  const { setToast } = useContext(ToastContext);
+  const { gameService } = useContext(Context);
+  const [
+    {
+      context: { state },
+    },
+  ] = useActor(gameService);
+
+  const inventory = state.inventory;
+
+  const sell = (amount = 1) => {
+    gameService.send("item.sell", {
+      item: selected.name,
+      amount,
+    });
+    setToast({
+      content: "SFL +$" + selected.sellPrice?.mul(amount).toString(),
+    });
+  };
+
+  const handleSellOne = () => {
+    sell(1);
+  };
+
+  const amount = new Decimal(inventory[selected.name] || 0);
+
+  return (
+    <div className="flex">
+      <div className="w-3/5 flex flex-wrap h-fit">
+        {Object.values(CAKES()).map((item) => (
+          <Box
+            isSelected={selected.name === item.name}
+            key={item.name}
+            onClick={() => setSelected(item)}
+            image={ITEM_DETAILS[item.name].image}
+            count={inventory[item.name]}
+          />
+        ))}
+      </div>
+      <OuterPanel className="flex-1 w-1/3">
+        <div className="flex flex-col justify-center items-center p-2 ">
+          <span className="text-shadow text-center">{selected.name}</span>
+          <img
```

Comment:
This image is very small here in comparison to the image inside the `Cook` section. I think they should be a similar size

_Sell_
<img width="100" alt="Screen Shot 2022-06-02 at 12 10 55 pm" src="https://user-images.githubusercontent.com/17863697/171533496-241987d0-0d29-445e-8475-123396401f3c.png">

_Buy_
<img width="100" alt="Screen Shot 2022-06-02 at 12 11 53 pm" src="https://user-images.githubusercontent.com/17863697/171533690-cfc09f03-ca09-4d7b-b843-1ddfc71d3801.png">